t/ui/16-tests_previous_results.t

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2016 SUSE Linux GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+BEGIN {
+    unshift @INC, 'lib';
+}
+
+use Mojo::Base -strict;
+use Test::More;
+use Test::Mojo;
+use OpenQA::Test::Case;
+
+OpenQA::Test::Case->new->init_data;
+
+use t::ui::PhantomTest;
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+
+my $get       = $t->get_ok('/tests/99946#previous')->status_is(200);
+my $tab_label = $t->tx->res->dom->at('li a[href=#previous]')->all_text;
+is($tab_label, q/Previous results (1)/, 'previous results with number is shown');
+my $previous_results_header = $t->tx->res->dom->at('#previous #scenario')->all_text;
+is($previous_results_header, q/Results for opensuse-13.1-DVD-i586-textmode, limited to 10/, 'header for previous results with scenario');
+$get->element_exists('#res_99945',                'result from previous job');
+$get->element_exists('#res_99945 .result_passed', 'previous job was passed');
```

Comment:
but past perfect is built with had. So it's either "the job had passed" as past perfect or "the job was passed" as past of be with the adjective 'passed'. So much about the grammar. But neither past perfect makes sense here as there is no other event - nor does past, as the job's state is passed even in the future. It won't change anymore. Anyway, I was just curious.